README.md

```diff
@@ -345,6 +345,18 @@ instance) and then getting its desired capacity.
 
 No options here besides `wait`.
 
+#### `query`
+
+A class method `query` is provided on the `Stack` class to find stacks matching certain criteria:
+
+* `regex`: only queries whose name matches this regex will be returned
+* `regions`: only stacks in these regions will be queried (defaults to US regions)
+* `active`: if true, excludes stacks that have been deleted or that failed to create
```

Comment:
I think we should add default values to the documentation

Reply:
Done.